### ISSUE TITLE
Use Unix TimeStamp instead

### ISSUE THREAD

**Issue by pauldeng** [u-blox/ubxlib_cellular_applications_xplr_iot#9]
Hi repo owner,

Appreciate your hard work.

I am testing this repo and I notice the payload has timestamp in the format like this `"Timestamp":"05:07:53.144"`.

It seems not very standard and would recommend to use Unix Timestamp in the unit of seconds like this `"Timestamp": 1692091468` which converts to `GMT | Tue Aug 15 2023 09:24:28 GMT+0000` according to https://www.unixtimestamp.com/.

Cheers,
Paul

**Comment by philwareublox** [comment]
Yes, I've named it "Timestamp", but really it's just "Time". I could change this to the Unix Timestamp. I'll think about this...

**Comment by pauldeng** [comment]
I try to fix this but confused, seems timestamp is a bit off. No idea where is the problem.

This is what I did, I added few lines to debug in registrationTask.c.
```c
        // remove the current local tick time from this
        // to obtain a rough "boot" time value.
        printf("time %" PRId64 "\n", time);
        printf("seconds %d \n", (uPortGetTickTimeMs() / 1000));
        unixNetworkTime = time - (uPortGetTickTimeMs() / 1000);
        printf("unixNetworkTime %" PRIu32 "\n", unixNetworkTime);
```

Then use TeraTerm to log local timestamp, here it is:
```log
[2023-08-17 16:16:58.870] 20243: Buttons #1 and #2 are now enabled
[2023-08-17 16:17:56.037] 77419: Connected to Cellular Network: Telstra Mobile (50501)
[2023-08-17 16:17:56.119] time 1692253123 // 16:18:43, not close to my wall clock
[2023-08-17 16:17:56.119] seconds 77 
[2023-08-17 16:17:56.119] unixNetworkTime 1692253046 //16:17:26, not close to my wall clock
```

None of `time` and `unixNetworkTime` are close to my wall clock.

Cheers,
Paul

**Comment by pauldeng** [comment]
I am trying to fix it myself, and I am poking around.
My network does not restrict NTP. So, which parameter you are holding the ntp synced timestamp?
Then it should be an easy fix.

**Comment by philwareublox** [comment]
I'm not sure what there is to fix :)
The code request the time from an NTP Server, and then removes the current uPortGetTickTimeMs() so that the logging timestamp is aligned.
What does the NTP Server return in your case, and what is the uPortGetTickTimeMs() report? 
The two values should give you a rough "boot" timestamp in the logging.

**Comment by pauldeng** [comment]
No, I was just trying to find out the var in your code which holds unix timestamp to replace the original time in mqtt payload.

**Comment by philwareublox** [comment]
The entire published message from each "appTask" is created in that appTask. The MQTTTask() which does the sending doesn't do anything with the payload - it just publishes it.

**Comment by pauldeng** [comment]
Understood, all I want to have is `"Timestamp": 1692091468`.
I was trying to fix it myself and sending you PR.
Obviously, I was reading the wrong part.

**Comment by philwareublox** [comment]
Just to note, UTC Time and GPS Time are not the same. GPS is ahead of UTC by 18 seconds. The timestamp from the NTPService will probably show a difference from the GNSS timestamp you get once you have a GNSS lock.
Although having said that, the GNSS POS() function in our GNSS devices returns the "UTC" timestamp :)
